src/test/java/com/synopsys/integration/jenkins/wrapper/JenkinsProxyHelperTest.java

```diff
@@ -126,48 +110,46 @@ private void noProxyInfoValidation(ProxyInfo proxyInfo) {
     }
 
     @ParameterizedTest
-    @MethodSource({ "validProxyInfo", "validProxyInfoWithNtlmDomain" })
-    public void testConstructorValidProxyInfo(String url, List<Pattern> ignoredProxyHosts, String testProxyUsername) {
-        ProxyInfo proxyInfo = new JenkinsProxyHelper(proxyHost, proxyPort, proxyUsername, proxyPassword, ignoredProxyHosts, testProxyUsername, ntlmWorkstation).getProxyInfo(url);
-        validProxyInfoValidation(proxyInfo, proxyHost, proxyPort, proxyUsername, proxyPassword, testProxyUsername, ntlmWorkstation);
+    @MethodSource({ "validProxyInfoInputTestData" })
+    public void testConstructorValidProxyInfo(String url, List<Pattern> ignoredProxyHosts, String testNtlmDomain) {
+        ProxyInfo proxyInfo = new JenkinsProxyHelper(expectedProxyHost, expectedProxyPort, expectedProxyUsername, expectedProxyPassword, ignoredProxyHosts, testNtlmDomain, expectedNtlmWorkstation).getProxyInfo(url);
+        validProxyInfoValidation(proxyInfo, testNtlmDomain, expectedNtlmWorkstation);
     }
 
     @ParameterizedTest
-    @MethodSource({ "validProxyInfo", "validProxyInfoWithNtlmDomain" })
-    public void testFromJenkinsValidProxyInfoJenkins(String url, List<Pattern> ignoredProxyHosts, String testNtlmDomain) {
-        String returnProxyUserName = (testNtlmDomain == null ? "" : ntlmDomain + "\\") + proxyUsername;
```

Comment:
returnProxyUserName  is no longer needed as an extra param is now supplied to provide the return data for Mockito.when.